renderer/src/shared/AutocompleteSelector/AutocompleteSelector.tsx

```diff
@@ -42,19 +42,25 @@ const AutocompleteSelector = ({
     return selectedItem.trim() === "" ? filteredList : [selectedItem, ...filteredList];
   }, [selectedItem, filteredList]);
 
-  const handleKey = (e) => {
-    if (!e.ctrlKey && !e.shiftKey && !e.metaKey && e.key === KEY_CODES.HOME) {
+  const handleKey: KeyboardEventHandler<HTMLInputElement> = (e) => {
+    if (!e.ctrlKey && !e.shiftKey && !e.metaKey && e.key === "Home") {
       e.preventDefault();
-      inputRef.current.selectionStart = 0;
-      inputRef.current.selectionEnd = 0;
+
+      if (inputRef.current) {
+        inputRef.current.selectionStart = 0;
+        inputRef.current.selectionEnd = 0;
+      }
     }
 
     if (!e.ctrlKey && !e.shiftKey && !e.metaKey && e.key === KEY_CODES.END) {
       e.preventDefault();
       const input = inputRef.current;
-      const length = input.value.length;
-      input.selectionStart = length;
-      input.selectionEnd = length;
+
+      if (input) {
+        const length = input.value.length;
+        input.selectionStart = length;
+        input.selectionEnd = length;
+      }
     }
 
     if (e.ctrlKey && e.key === KEY_CODES.ENTER) {
```

Comment:
I understand that these are not your changes, but on lines 71 and 81 on "e.code" there is an error of the types "Property 'code' does not exist on type 'KeyboardEvent<HTMLInputElement>'"

Reply:
It's not actual.